packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -87,8 +87,9 @@ contract TaikoL1 is EssentialContract, ITaikoL1, TaikoEvents, TaikoErrors {
 
         (meta_, deposits_) = LibProposing.proposeBlock(state, tko, config, this, _params, _txList);
 
-        if (!state.slotB.provingPaused) {
-            LibVerifying.verifyBlocks(state, tko, config, this, config.maxBlocksToVerifyPerProposal);
+        uint256 oddBase = config.maxBlocksToVerify >> 1;
+        if (oddBase != 0 && meta_.id % oddBase == 0 && !state.slotB.provingPaused) {
```

Comment:
why this code with oddBase here and in the proveBlock? I don't think I understand why this just doesn't do the modulo directly with some value.